### PR TITLE
Add Dockerfile for rethinkdb @ 2.2.4

### PR DESCRIPTION
**Pull request by alexcouper**

### CODE REVIEW

**Comment by alexcouper** [comment]
This is already released and in use by https://github.com/ployst/ployst/pull/9

File: rethinkdb/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+# limitations under the License.
+
+FROM rethinkdb:2.2.4
+
+MAINTAINER BinZhao <wo@zhaob.in>
+
+RUN apt-get update && \
+    apt-get install -yq curl && \
+    rm -rf /var/cache/apt/* && rm -rf /var/lib/apt/lists/* && \
+    curl -L http://stedolan.github.io/jq/download/linux64/jq > /usr/bin/jq && \
```
Comment (txels) [inline rethinkdb/Dockerfile]: Ah so this is the *sed of JSON data* I was wondering existed.
I'd rather us not install software straight off curl http downloads in production though.
Could we `sudo apt-get install jq` instead? https://stedolan.github.io/jq/download/

Comment (txels) [inline rethinkdb/Dockerfile]: Also technically this way of installing dependencies makes them moving targets.
Could we aim for having all dependencies pinned in our prod dockerfiles?

File: rethinkdb/run.sh [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+  # try to pick up first different ip from endpoints
+  IP=$(curl -s ${URL} --cacert /var/run/secrets/kubernetes.io/serviceaccount/ca.crt --header "Authorization: Bearer ${token}" \
+    | jq -s -r --arg h "${MYHOST}" '.[0].subsets | .[].addresses | [ .[].ip ] | map(select(. != $h)) | .[0]') || exit 1
+  [[ "${IP}" == null ]] && IP=""
+fi
+
+if [[ -n "${IP}" ]]; then
+  ENDPOINT="${IP}:29015"
+  echo "Join to ${ENDPOINT}"
+  exec rethinkdb --bind all  --join ${ENDPOINT}
```
Comment (txels) [inline rethinkdb/run.sh]: You're right this is quite nice.

File: rethinkdb/Dockerfile (hunk 1 above)
Comment (txels) [inline rethinkdb/Dockerfile]: Ooh, gotta love this tool:

```
$ cat ployst-production/kubernetes/postgres/pg-rc.json | jq ".spec.template.spec.volumes[].name"
"persistent-storage"
```

Comment (alexcouper) [inline rethinkdb/Dockerfile]: I've used 1.5 (still curling it though as whatever rethinkdb is using seems to only have 1.4 available)

**Pull request merged**